src/test/java/ClientTest.java

```diff
@@ -43,20 +41,20 @@ public void setUp() {
             ex.printStackTrace();
         }
 
-        Credentials credentials = new Credentials(
-            this.config.getProperty("clientId"),
-            this.config.getProperty("clientSecret"),
-            this.config.getProperty("apiKey"),
-            this.config.getProperty("authEndpoint"),
-            this.config.getProperty("apiEndpoint")
-        );
+        Credentials credentials;
+            credentials = new Credentials(
```

Comment:
Do we need to declare credentials before defining it? Also the indentation looks incorrect.